files/json_output_formatter.rb

```diff
@@ -15,19 +15,24 @@ def dump_summary(summary)
       map { |example| example.metadata[:points].to_i }.
       sum
 
+    score = (earned_points.to_f / total_points).round(4)
+    if score.nan?
+      score = 0
```

Comment:
I was about to suggest 

```suggestion
      score = 0 if score.nan?
```

but I never did really like that style; I prefer this more straightforward version. But thought I would mention it, since it occurred to me.

Reply:
I tend to avoid single line syntax, but this might be a clear enough line that it makes sense to use it.

Reply:
I agree with you, and my follow-up to the discussion below shows why; it so often happens that I end up wanting to expand the branch with more logic.